pandas/io/common.py

```diff
@@ -188,6 +185,16 @@ def is_gcs_url(url) -> bool:
         return False
 
 
+def urlopen(*args, **kwargs):
+    """
+    Lazy-import wrapper for stdlib urlopen, as that imports a big chunk of
+    the stdlib.
+    """
+    from urllib.request import urlopen as _urlopen
```

Comment:
I think the convention is trailing underscores to prevent clashes.

Or, 

```python
import urllib.request

return urllib.request.urlopen(*args, **kwargs)
```